internal/commands/builder_create_test.go

```diff
@@ -92,6 +92,20 @@ func testCreateCommand(t *testing.T, when spec.G, it spec.S) {
 			})
 		})
 
+		when("--pull-policy is not specified", func() {
```

Comment:
Would prefer to add some more thorough testing, but it seems the tests in this file are only for checking errors?